modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -49,12 +44,11 @@
 	--n-tabs-title-padding-left: var( --n-tabs-default-padding-inline );
 	--n-tabs-title-border-radius: initial;
 	--n-tabs-title-transition: 0.3s;
-
 	--n-tabs-icon-color: var( --e-global-color-secondary, var( --n-tabs-color-secondary-fallback ) );
 	--n-tabs-icon-color-hover: var( --n-tabs-title-color-hover );
 	--n-tabs-icon-color-active: #{$white};
 	--n-tabs-icon-gap: 5px;
-
+	width: 100%;
```

Comment:
nice and simple fix. not much to say here (code wise) :)

Reply:
Aviv has tested it functionality wise. He has approved this concept.

Reply:
Lovely - he is the gold standard of testing 👑